sample/SampleApi/Program.cs

```diff
@@ -12,7 +12,7 @@
 builder.Services.AddSingleton<SendDataHandler>();
 
 // Register Atc.Azure.Messaging dependencies
-builder.Services.ConfigureMessagingServices(builder.Configuration);
+builder.Services.ConfigureMessagingServices(builder.Configuration); // ConfigureMessagingServices(builder.Configuration, true)
```

Comment:
Why is there a `//` at the end of line 15 ?

Reply:
fixed